sql/sql_yacc.yy

```diff
@@ -1236,6 +1239,7 @@ void warn_about_deprecated_national(THD *thd)
 %token<keyword> GTID_SYM
 %token<keyword> GTID_EXECUTED                 /* MYSQL */
 %token<keyword> SUPER_READ_ONLY_SYM           /* MYSQL */
+%token  DB_METADATA_SYM
```

Comment:
8.0 has some attempts to have parser token ID (and resulting query digest) stability, please check: might be a good idea to skip 100 IDs and start a FB-specific section

Reply:
The 3 tokens above this are also added by facebook - I simply added this one to the end of the existing list.

Reply:
Are you sure? All three look like MySQL ones

Reply:
There is a comment at line 1236: "Tokens from FB MySQL"

Reply:
You are right, sorry. And there is no provision to skip IDs, thus everything already is as good as it could be